src/librbd/cache/pwl/ImageCacheState.cc

```diff
@@ -63,14 +62,24 @@ ImageCacheState<I>::ImageCacheState(
   std::istringstream iss(f["pwl_size"]);
   iss >> pwl_size;
   size = pwl_size;
+}
 
-  // Others from config
-  ConfigProxy &config = image_ctx->config;
-  log_periodic_stats = config.get_val<bool>("rbd_persistent_cache_log_periodic_stats");
+template <typename I>
+void ImageCacheState<I>::update_timestamp(void) {
+  struct timespec pwl_timestamp;
+  clock_gettime(CLOCK_REALTIME, &pwl_timestamp);
+  if (pwl_timestamp.tv_sec != 0) {
+    time_t ts = pwl_timestamp.tv_sec;
+    update_time = ctime(&ts);
+    update_time = update_time.substr(0, update_time.length() - 1);
```

Comment:
The timestamp should be encoded as an integer, whole second precision should be sufficient: `stats_timestamp.sec()`.  This conversion to string is to be done on the `rbd status` side.